docs/go-hello-world.md

```diff
@@ -37,7 +39,7 @@ Then, add the Temporal Go SDK as a project dependency:
 go get go.temporal.io/sdk@latest
 ```
 
-## ![](/img/docs/apps.png) &nbsp;&nbsp; "Hello World!" app
+## ![](https://raw.githubusercontent.com/temporalio/documentation-images/main/static/apps.png) "Hello World!" app
```

Comment:
I guess this doesn't work for initial local development?

Reply:
It can, the images are in a separate repo (documentation-images). So, you can put up a quick pr with the images and link to them that way. If the branch merges, then you change commit/branch name to main in the url.
But just another option for image storage for the docs long term. Shawn mentioned he likes having the images hosted on github and initially recommended creating issues and dragging and dropping there. I think that way would get too messy for us long term, so trying it this way.